components/console/src/core/domain/entities/organization-avatar-entity.ts

```diff
@@ -0,0 +1,4 @@
+export interface OrganizationAvatarEntity {
+  organizationId: string
+  imageBase64: string
```

Comment:
I don't think this should be the way to go
The image itself should be embedded into the OrganizationEntity directly, instead of creating a new Entity just for this infra detail.